### PR TITLE
Release notes for 2024.1

### PR DESCRIPTION
**Pull request by sdrave**

### CODE REVIEW

**Comment by sdrave** [comment]
@pymor/all please add text as you see fit.

File: docs/source/release_notes/2024.1.rst [hunk 1]
```diff
@@ -0,0 +1,71 @@
+pyMOR 2024.1 (July 5, 2025)
+-------------------------------
+
+We are proud to announce the release of pyMOR 2024.1!
+This release features ...
+
+pyMOR 2024.2 contains contributions by Maximilian Bindhak and Art Pelling.
```
Comment (artpelling) [inline docs/source/release_notes/2024.1.rst:7]:
```suggestion
pyMOR 2024.1 contains contributions by Maximilian Bindhak and Art Pelling.
```

**Review by artpelling — DISMISSED** [review]

File: docs/source/release_notes/2024.1.rst (hunk 1 above)
Comment (sdrave) [reply docs/source/release_notes/2024.1.rst:7]: Fixed.

File: docs/source/release_notes/2024.1.rst [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+Main new features
+^^^^^^^^^^^^^^^^^
+
+Vector fitting
+~~~~~~~~~~~~~~
+We added an implementation of vector fitting
+`[#2214] <https://github.com/pymor/pymor/pull/2214>`_,
+a least-squares data-driven method for frequency-response data of linear
+time-invariant systems.
+This extends to the available methods:
```
Comment (HenKlei) [inline docs/source/release_notes/2024.1.rst:32]:
```suggestion
This is an extension to the available methods:
```
I am not sure what is meant here...

File: docs/source/release_notes/2024.1.rst [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+
+* vector fitting
+
+* successive constraints method
+
+* shifted Cholesky QR algorithm
+
+* improved randomized range finder
+
+* additional tutorials and MOR methods overview
```
Comment (HenKlei) [inline docs/source/release_notes/2024.1.rst:16]: We never had this list before but instead stated how many commits entered the release. I am fine with both versions actually.

File: docs/source/release_notes/2024.1.rst [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+a least-squares data-driven method for frequency-response data of linear
+time-invariant systems.
+This extends to the available methods:
+interpolatory Loewner method (:class:`~pymor.reductors.loewner.LoewnerReductor`)
+and mixed greedy/least-squares AAA algorithm
+(:class:`~pymor.reductors.aaa.PAAAReductor`).
+
+Successive constraints method
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We implemented the successive constraints method,
```
Comment (HenKlei) [inline docs/source/release_notes/2024.1.rst:39]:
```suggestion
We implemented the successive constraints method in order
```

File: docs/source/release_notes/2024.1.rst [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We added an implementation of the shifted Cholesky QR algorithm
+`[#2177] <https://github.com/pymor/pymor/pull/2177>`_,
+which has potential to exploit higher performance linear algebra routines
+compared to the Gram-Schmidt algorithm
+(:func:`~pymor.algorithms.gram_schmidt.gram_schmidt`).
+
+Improved randomized range finder
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The :mod`~pymor.algorithms.rand_la` module has seen various improvements.
```
Comment (HenKlei) [inline docs/source/release_notes/2024.1.rst:57]:
```suggestion
The :mod:`~pymor.algorithms.rand_la` module has seen various improvements.
```

File: docs/source/release_notes/2024.1.rst [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+`solution_d_mu` option of `compute` now returns a dict of the form
+`{(parameter, index): sensitivity}` instead of appending solution sensitivities
+for different parameter indices to a single array. `output_d_mu` now always
+returns a frozen dict-like object of the form `{(parameter, index): sensitivity}`,
+where `sensitivity` is a 2D NumPy array with axis 0 corresponding to time and
+axis 1 corresponding to output component. The returned object has a `to_numpy`
+method to convert it to a single array. The `return_array` parameter has been
+removed from `output_d_mu`.
+
+Only compute keeps `**kwargs`, which are only allowed to be used to select
```
Comment (HenKlei) [inline docs/source/release_notes/2024.1.rst:97]:
```suggestion
Only `compute` keeps `**kwargs`, which are only allowed to be used to select
```

**Review by HenKlei — COMMENTED** [review]
Looks good!

**Pull request merged**